<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>623</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>769</width>
    <height>623</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>APGB</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabs">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>769</width>
      <height>622</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>769</width>
      <height>622</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_editor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>Editor</string>
     </attribute>
     <widget class="QPushButton" name="btn_import">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>540</y>
        <width>88</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Import</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_obj0">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>OBJ0         :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_window_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>470</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_window_0">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>470</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_obj0_3">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>390</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_bg_1">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>350</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_window_3">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>470</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_obj1_0">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>430</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_bg_3">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>350</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_bg_3">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>350</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_bg_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>350</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_bg_1">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>350</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_obj1_1">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>430</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_obj1_3">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>430</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_save">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>540</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_obj1_1">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>430</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_window">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>WINDOW :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_editor_tip">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>320</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Please use 24-bit hexdecimal color values</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_obj0_1">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>390</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_obj0_1">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>390</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_obj0_0">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>390</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_scale">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>510</y>
        <width>531</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>darkest &lt;-----------------------------------------------------------------------------------------&gt; brightest</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_obj0_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>390</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_window_3">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>470</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_bg_0">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>350</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_bg_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>350</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_obj1_0">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>430</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_obj1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>OBJ1         :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_window_0">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>470</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_window_1">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>470</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_obj0_3">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>390</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_obj1_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>430</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_window_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>470</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_obj0_0">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>390</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_obj1_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>430</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_obj1_3">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>430</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_obj0_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>390</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_bg_0">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>350</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_window_1">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>470</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_bg">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>BG              :</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="view_img">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>320</width>
        <height>288</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>0</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sceneRect">
       <rectf>
        <x>0.000000000000000</x>
        <y>0.000000000000000</y>
        <width>0.000000000000000</width>
        <height>0.000000000000000</height>
       </rectf>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_img_next">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>160</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Advance to next image.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>|&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_img_prev">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Go back to previous image.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;|</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_flip_window">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>470</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Flip WINDOW intensities.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;|&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_flip_obj0">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>390</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Flip OBJ0 intensities.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;|&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_flip_obj1">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>430</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Flip OBJ1 intensities.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;|&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_flip_bg">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>350</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Flip BG intensities.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;|&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_flip_all">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>310</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Flip all intensities for all palettes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Flip All</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_populate_all">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Populate all intensities for each palette based on BG intensities. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Populate All</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="btn_dot_matrix">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>20</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Display image with a dot matrix filter.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Dot Matrix</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_converter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>Convert</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>641</width>
        <height>246</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="lbl_convert_title">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Convert Palette</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lbl_load">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Source:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="txt_convert_load">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_get_load">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lbl_type">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Format:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="r_csv">
           <property name="text">
            <string>CSV</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="r_gpl">
           <property name="text">
            <string>GIMP GPL (v2)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="r_jasc">
           <property name="text">
            <string>PAL(JASC)</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignHCenter">
          <widget class="QRadioButton" name="r_hex_txt">
           <property name="text">
            <string>HEX (Text)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lbl_save">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Destination:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="txt_convert_save">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_get_save">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item alignment="Qt::AlignmentFlag::AlignRight">
          <widget class="QPushButton" name="btn_convert_reset">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignLeft">
          <widget class="QPushButton" name="btn_convert_save">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
